analog-delivery-monitoring/source-views/matriceCostiPivot.sql

```diff
@@ -25,7 +25,7 @@ $QueryMetadata
 CREATE OR REPLACE TEMPORARY VIEW matrice_costi_2024
 USING csv
 OPTIONS (
-  path "s3a://${CORE_BUCKET}/external/matrice_costi/matrice_costi_202408_pivot.csv.gz",
+  path "s3://${CORE_BUCKET}/external/matrice_costi/matrice_costi_202408_pivot_v202410.csv.gz",
```

Comment:
@francesca-restante il protocollo va lasciato in "s3a"

Reply:
Aggiornato nella commit successiva